.github/workflows/beta_codesync.yml

```diff
@@ -0,0 +1,31 @@
+name: sync up to codecommit in beta
+
+on:
+  push:
+    tags-ignore:
+      - '*'
+    branches:
+      - '*'
+
+jobs:
+  betasync:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::377429403256:role/github-codesync-role
```

Comment:
I don't see this role created for the account. What permissions are attached to it and how are we limiting access only to this github action?

Reply:
Nvm found it.

Reply:
This role is created by an internal code commit. You can find it under our code repo. And it has limited to only github can assume it and push code to `GitHubServiceConnectAgent`.